embedchain/embedchain.py

```diff
@@ -47,11 +47,12 @@ def __init__(self, db=None, ef=None):
         self.collection = db.collection
         self.user_asks = []
 
-    def _get_loader(self, data_type):
+    def _get_loader(self, data_type, url=None):
```

Comment:
no need for extra `url` leave it to the loader to decide what to do with url that are local file